public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
+          fields: {
+            '*': {
+              fragment_size: 0,
```

Comment:
Elasticsearch uses the fragment_size parameter to determine how much context to provide around a highlight. When set to a value greater than 0, ES will return that many characters around the highlighted term to provide context. Since I only want to show the highlighted terms, with no context, I set this value fragment_size to 0.